### DISCUSSION TITLE
Roland A-49 works

### DISCUSSION THREAD

**Discussion by z80cpu** [probonopd/MiniDexed#476] [Q&A] [answered]
Recently I have built the Minidexed and it's cool working with akai mini play, but sadly with my Roland A-49 it doesn't work over USB. Please help!

**Comment by craigyjp** [comment]
Is the A49 class compliant as a usb device or does it need drivers to work
on your PC?

**Reply by Banana71** [reply]
Roland offers drivers for the Roland A-49 on their website.

**Comment by craigyjp** [comment]
Sounds like it isn't compliant then and won't work with MiniDexed or
anything that needs a class compliant device like a Teensy usb host port.

**Comment by z80cpu** [comment, marked as answer]
Thanks guys for cooperation. I found how to enable  compliant mode: 
1. Press FUNCTION so it is lit.

2. Press the key with the label "ADV" above it.

3. Use the + / -  buttons to make the setting:

    + (lit): Advanced Driver - this setting requires a Roland USB driver to be installed on the computer.
    - (lit): Generic Driver - this setting uses the driver in your computer's operating system.

4. Re-start the A-49 to finalize the setting.

With generic driver mode all working good!

**Reply by Banana71** [reply]
Great that it still worked.  It would be helpful if you explained that again in this post. #279
